tests/phpunit/unit-tests/api/utilities/beansSanitizePath.php

```diff
@@ -29,21 +29,102 @@ protected function setUp() {
 		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
 	}
 
+	/**
+	 * Test beans_sanitize_path() should remove Windows drive.
+	 */
+	public function test_should_remove_windows_drive() {
+		$this->assertSame( '/Program Files/tmp/', beans_sanitize_path( 'C:\Program Files\tmp\\' ) );
+		$this->assertSame( '/Foo/bar/baz/index.txt', beans_sanitize_path( 'D:\Foo\bar\baz\index.txt' ) );
+	}
+
 	/**
 	 * Test beans_sanitize_path() should sanitize for filesystem.
 	 */
 	public function test_should_sanitize_for_filesystem() {
-		$this->assertSame( BEANS_TESTS_DIR, beans_sanitize_path( BEANS_TESTS_DIR ) );
-		$this->assertSame( __DIR__, beans_sanitize_path( __DIR__ ) );
-		$this->assertSame( BEANS_TESTS_DIR, beans_sanitize_path( __DIR__ . '/../../../' ) );
-		$this->assertSame( BEANS_TESTS_DIR . '/bootstrap.php', beans_sanitize_path( __DIR__ . '/../../../bootstrap.php' ) );
+		$this->assertSame( $this->prepare_path( BEANS_TESTS_DIR ), beans_sanitize_path( BEANS_TESTS_DIR ) );
+		$this->assertSame( $this->prepare_path( __DIR__ ), beans_sanitize_path( __DIR__ ) );
+
+		// test phpunit's path.
+		$this->assertSame(
+			$this->prepare_path( BEANS_TESTS_DIR ) . '/bootstrap.php',
+			beans_sanitize_path( BEANS_TESTS_DIR . DIRECTORY_SEPARATOR . 'bootstrap.php' )
+		);
+		$this->assertSame( $this->prepare_path( __FILE__ ), beans_sanitize_path( __FILE__ ) );
+
+		// test beans' theme root path.
+		$this->assertSame(
+			$this->prepare_path( rtrim( BEANS_ROOT_DIR, DIRECTORY_SEPARATOR ) ),
+			beans_sanitize_path( BEANS_ROOT_DIR )
+		);
+		$directory_separator = '\\' === DIRECTORY_SEPARATOR ? '/' : '';
+		$this->assertSame(
+			$this->prepare_path( BEANS_ROOT_DIR ) . $directory_separator . 'functions.php',
+			beans_sanitize_path( BEANS_ROOT_DIR . 'functions.php' )
+		);
 	}
 
 	/**
-	 * Test beans_sanitize_path() should remove Windows drive.
+	 * Test beans_sanitize_path() should sanitize for filesystem symbolic links.
 	 */
-	public function test_should_remove_windows_drive() {
-		$this->assertSame( '/Program Files/tmp/', beans_sanitize_path( 'C:\Program Files\tmp\\' ) );
-		$this->assertSame( '/Foo/bar/baz/index.txt', beans_sanitize_path( 'D:\Foo\bar\baz\index.txt' ) );
+	public function test_should_sanitize_for_filesystem_symbolic_links() {
```

Comment:
The `beans_sanitize_path` function also converts the symbolic links of `/../` into an absolute path.  Let's test that separately and purposefully to ensure it works on Mac and Windows.